lib/mocha/class_method.rb

```diff
@@ -102,6 +95,11 @@ def method_visibility
 
     attr_reader :original_method, :original_visibility
 
+    def original_method_body
+      original_method_in_scope = original_method
+      PRE_RUBY_V19 ? proc { |*args, &block| original_method_in_scope.call(*args, &block) } : original_method
```

Comment:
Minor: I'm not super-keen on using the ternary operator here, because the line ends up being quite long.